keras_retinanet/callbacks/eval.py

```diff
@@ -67,6 +76,4 @@ def on_epoch_end(self, epoch, logs=None):
         logs['mAP'] = self.mean_ap
 
         if self.verbose == 1:
-            for label, average_precision in average_precisions.items():
-                print(self.generator.label_to_name(label), '{:.4f}'.format(average_precision))
```

Comment:
Printing the mAP per label can be quite useful. Why did you remove it?

Reply:
It is not removed, I just moved it inside the for loop at line 60

Reply:
I see, then I haven't said anything :)